.github/workflows/build-engine-container.yml

```
@@ -221,15 +279,36 @@ jobs:
         id: buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: <Setup> Login to DockerHub
+      - name: <Setup> Login to GitHub Container Registry (ghcr.io)
         uses: docker/login-action@v3
         with:
-          username: ${{ vars.DOCKERHUB_USERNAME }}
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: <Setup> Login to Docker Hub
+        if: needs.config.outputs.dockerhub_enabled == 'true'
+        uses: docker/login-action@v3
+        with:
+          username: ${{ needs.config.outputs.dockerhub_username }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
-      - name: <Build/Deploy> Build and Deploy multi-platform Docker image
+      - name: <Build/Deploy> Build and Deploy multi-platform Docker image for ghcr.io
+        env:
+          IMAGE_NAME: ${{ needs.config.outputs.ghcr_repository }}
+          VERSION_OR_LATEST: ${{ needs.config.outputs.version_or_latest }}
+          AMD64_IMAGE_PREFIX: ${{ matrix.amd64_image_prefix }}
+          ARM64_IMAGE_PREFIX: ${{ matrix.arm64_image_prefix }}
+          MULTI_PLATFORM_IMAGE_PREFIXES: ${{ matrix.multi_platform_image_prefixes }}
+        run: |
+          bash ./tools/build_and_push_multi_platform_docker_images.bash
+
+      # TODO: Docker Hubに単一プラットフォームイメージのタグを作成しないようにしたい
+      # そのためには、宛先レジストリにタグを作成せずに、レジストリ間でマニフェストリストをコピーする必要があり、難しいので保留
```

（細かいけど）

省けるところは省いちゃうとスッキリ！
```suggestion
      # 宛先レジストリにタグを作成せずにレジストリ間でマニフェストリストをコピーする必要があり、難しいので保留中
```

---

Suggestion適用します。